docs/content/FormControl.mdx

```diff
@@ -0,0 +1,414 @@
+---
+componentId: formControl
+title: FormControl
+status: Alpha
+description: The FormControl component is used to render a labelled text input and, optionally, associated validation text and hint text.
+source: https://github.com/primer/react/blob/main/src/FormControl/FormControl.tsx
+storybook: '/react/storybook?path=/story/forms-inputfield--text-input-field'
+---
+
+import {FormControl, TextInputWithTokens, Autocomplete, Select, Textarea, Checkbox, Radio} from '@primer/react'
+import {MarkGithubIcon} from '@primer/octicons-react'
+
+## Examples
+
+### Basic
+
+```jsx live
+<FormControl>
+  <FormControl.Label>Name</FormControl.Label>
+  <TextInput />
+</FormControl>
+```
+
+### `variant="stack"` (default) inputs
```

Comment:
```variant="stack" (default) inputs``` didn't make sense to me at first because it wasn't clear to me what `variant="stack"` meant. How about this?

```suggestion
### With complex inputs
```